gulpfile.js

```diff
@@ -258,6 +262,30 @@ gulp.task(
 	}
 );
 
+gulp.task(
+	'blocks',
+	function( done ) {
+		FOLDERS.map(
+			function( folderConfig ) {
+				var folder = folderConfig.folder;
+				var PATHS  = folderConfig.PATHS;
+				var MATCH  = folderConfig.MATCH;
+				var SRC    = JSON.parse( JSON.stringify( folderConfig.SRC.js ) );
+				var baseSRC = path.join( folder, PATHS.blocksSource );
+				SRC.unshift( path.join( baseSRC, MATCH.restrictJS ) );
```

Comment:
Is this restrictJS variable meant to only use (or not use) files on the root of the blocksSource folder?

I'm not clear on what this does, can you elaborate a bit? Thanks

Reply:
I called it "restrict" to restrict files to be compiled to the root of **blocksSource** folder. Otherwise, any module placed in a subfolder would be compiled too. So, the restrictJS is `*.JS` instead of using `**/*.js` needed here.

Reply:
Maybe we can call it nonRecursiveJS then?